.github/workflows/config.yml

```diff
@@ -3,7 +3,7 @@ on: [push]
 jobs:
   install-gemmini:
     name: gemmini-install
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
     container:
       image: ucbbar/chipyard-ci-image:3f9150
       options: --entrypoint /bin/bash
```

Comment:
Once you switch to self-hosted the container doesn't matter anymore

Reply:
Ok. I'll revert the change to switch to self-hosted